benchmark-result-linux-with-server.md

```diff
@@ -0,0 +1,5 @@
+| Command | Mean [s] | Min [s] | Max [s] | Relative |
```

Comment:
I know these generated files will cause issues with linting, but I'm not sure if we want to commit them or not.
On one hand it will be easier to reference/point to the data, on the other it is generated and thus not needed to be part of the repo.